debian/usr/share/applications/D1GraphicsTool.desktop

```diff
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=0.5.0
+Version=1.0.0
```

Comment:
I don't think this is actually has any benefits and with out it we will have one less things we have to remember to update on release.
```suggestion
```

Reply:
I think you just reverted it rather then remove the line, i did so here: https://github.com/diasurgical/d1-graphics-tool/commit/1090703a696cd57e10892e1bf727ec2461a6f960

Reply:
Ah right, my bad, didn't see it's line removal on commit change rather than character change.